docs/apache-airflow-providers-apache-spark/connections/spark.rst

```diff
@@ -19,15 +19,15 @@
 
 .. _howto/connection:spark:
```

Comment:
Lets rename the file as then to spark-submit.rst

Reply:
renamed :)